TeamCode/src/main/java/t10/bootstrap/BootstrappedOpMode.java

```diff
@@ -59,10 +59,10 @@ public void dumpEnvironment() {
      * Waits a desired number of milliseconds.
      * <strong>{@link Thread#sleep(long)} does not work, use this instead!</strong>
      *
-     * @param milliseconds Number of milliseconds to wait
+     * @param seconds Number of seconds to wait
      */
-    public void sleep(long milliseconds) throws InterruptedException {
-        SystemClock.sleep(milliseconds);
+    public void sleep(double seconds) {

```

Comment:
check to make sure that sleep(**LONG**) is not being used. tell me if there are any usages of sleep(**LONG**). just so code doesn't start breaking because they're now being interpreted as seconds instead of milliseconds